tests/Makefile

```diff
@@ -0,0 +1,29 @@
+all: lint unit_tests
+
+lint:
```

Comment:
@smarnach 

> If we use wget to download the file from the instance manager each time the tests are run then we can break the tests in this repo by making a change to the other repo. I don't think we want this, so instead I suggest to add a make target that updates the files from the instance manager. We will have to do this manually, but nothing bad happens if we don't do it, and it prevents our tests from randomly breaking.

Yup, adding a make target seem fine yep - that would actually be in line with how we update dependencies. 

> The pylintrc from the instance manager can't be used unmodified, since it includes load-plugins=lint_django, which won't work for this repo. I tried overriding this from the command line, but the plugins given on the command line are loaded in addition to those in the configuration file, so there is nothgin I can do. This is another reason why we can't download the file from the instance manager.

With the make target, that could be something to `sed` out of the file you retrieve?

> I can't make prospector work when using the same configuration as the instance manager. I runs fine and claims that all is good, but if I modify the files to contain pylint violations, they don't show up in the output. If I run pylint manually, all works well, and when I add violations of e.g. pyflakes I can see them as well. Prospector doesn't offer any debug output, so it's very hard to tell what's going on. It claims to use the correct pylint configuration file, but I can't see any indication that pylint is actually being run. The next step here would be to dig into the prospector code to figure out what it's actually doing.
> How much additional time am I supposed to invest in this? I personally don't think the change to use exactly the same configuration adds much value over the current version, which is already mostly consistent with the instance manager, and we only have like 200 lines of code in this repo.

Since the LB is pretty much the first service outside of the instance manager repo, you're pretty much hitting all the bottlenecks; we should actually have thought of this during the discovery, I don't think we really planned for this. If that's some comfort, that will make it much easier for the next services to be setup, once you'll have done it once for here. 

That also makes it important to sort out now - it's precisely because it's only 200 lines of code for now that it's useful to do it now. If it grows, or when other services will be setup, it will be a higher barrier. Especially with things like linting, where adding it or changing it late can be especially painful/lengthy.

I would say it's worth digging a bit to get it to run. I don't want to make this spillover one more time though, so if you can't figure out why prospector doesn't run in the time you have available, I would be ok with creating a task for it on the scaling epic, but if we can avoid adding another tech debt story to the backlog that would be good.